lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -335,6 +346,7 @@ public function errorInfo()
     public function closeCursor()
     {
         $this->_stmt->free_result();
+        $this->_rowValuesAreBound = false;
```

Comment:
Not sure. It is just moving store_result() scares me alot. It history it have been a very picky call to get right, with all the different drivers and such.